src/Components/Calculator.js

```diff
@@ -1,121 +1,89 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      element: {
-        total: null,
-        next: null,
-        operation: null,
-      },
-    };
-    this.result = this.result.bind(this);
-  }
+const Calculator = () => {
+  const [element, AddNewEl] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
 
-  result(event) {
-    const { element } = this.state;
+  const Calc = (event) => {
     const NewEl = calculate(element, event.target.textContent);
```

Comment:
[Optional] CamelCase should be used to write the variable in JavaScript. In order to make your code look more professional, it would be good to change "NewEl" to "newEl".